### PR TITLE
#373 & adding some other missing TC items in release history file.

### PR DESCRIPTION
**Pull request by harleenkohli**

### CODE REVIEW

File: src/Sarif/Schemata/sarif-schema.json [hunk 1]
```diff
@@ -2699,6 +2699,11 @@
           ]
         },
 
+        "justification": {
```
Comment (michaelcfanning) [inline src/Sarif/Schemata/sarif-schema.json]: >justification [](start = 9, length = 13)

when was this proposed? i did not participate in the conversation. :) i don't want this data in the log file, personally.

**Comment by lgolding** [comment]
Approving after discussion with and at request of @mikefan@microsoft.com

**Comment by harleenkohli** [comment]
thanks so much Larry!

---
In reply to: [485005470](https://github.com/Microsoft/sarif-sdk/pull/1444#issuecomment-485005470) [](ancestors = 485005470)

File: src/Sarif/Schemata/sarif-schema.json (hunk 1 above)
Comment (harleenkohli) [reply src/Sarif/Schemata/sarif-schema.json]: marking this by design based on chat conversation.

---
In reply to: [277075812](https://github.com/Microsoft/sarif-sdk/pull/1444#discussion_r277075812) [](ancestors = 277075812)

**Pull request merged**